iohub/cli/cli.py

```diff
@@ -31,3 +33,52 @@ def print_reader_info(reader):
     print(f"Channel names:\t {reader.channel_names}")
     print(f"Z step (um):\t {reader.z_step_size}")
     print("")
+
+
+@cli.command()
+@click.help_option("-h", "--help")
+@click.option(
+    "--input",
+    "-i",
+    required=True,
+    type=click.Path(exists=True, file_okay=False, resolve_path=True),
+    help="Input Micro-Manager TIFF dataset directory",
+)
+@click.option(
+    "--output",
+    "-o",
+    required=True,
+    type=click.Path(exists=False, resolve_path=True),
+    help="Output zarr store (/**/converted.zarr)",
+)
+@click.option(
+    "--format",
+    "-f",
+    required=False,
+    type=str,
+    help="Data type, 'ometiff', 'ndtiff', 'singlepagetiff'",
```

Comment:
Consider "in-format"? As written it's not clear if this is the input or the output type.

Reply:
I think the help message says 'convert from many TIFFS to OME-Zarr'. But happy to change if that is not clear enough.

Reply:
Fair point. I have some followup questions after thinking more:

- is the `format` option necessary? Do you think we could reuse the dispatching logic from `imread`?
- is `iohub convert` too strong a name? Is `iohub tiff2zarr` too specific?

Reply:
The format option is for
1. when `imread` detection fails (I suspect that it might happen with some legacy MM single-page TIFF, but nite tested)
2. when `imread` does not have auto detection for a certain format, e.g. uPTI acquisition.

Reply:
> * is `iohub convert` too strong a name? Is `iohub tiff2zarr` too specific?

I just copied the `recOrder` option over, but I agree that for now`tiff2zarr` is a more accurate description. Not sure if we will have other formats added in the near future though, if so the `convert --format <fmt>` can make a more stable API. One potential source I can think of is the legacy NGFF v0.1 datasets.

Reply:
KK sounds good. I'm flexible on the renaming...you can decide.